src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx

```diff
@@ -33,7 +29,7 @@ We've integrated discussion tools directly into New Relic so you can:
 * **Create shared knowledge and history.** Collaborating within the New Relic platform allows your team to build a searchable body of shared knowledge.
 ## Comment on any page [#comment-on-any-page]
 
-On the global navigation, there's a new “add comment” icon on most pages. A green dot on the icon means there's an open discussion on the page, while a gray dot indicates a closed discussion that may have information to help resolve a current issue. 
+On the global navigation, there's an “add comment” icon on most pages. A green dot on the icon means there's an open discussion on the page, while a gray dot indicates a closed discussion that may have information to help resolve a current issue. 
```

Comment:
Per style guide, we bold UI elements and use the verbiage as it appears in the UI. So this should be stylized as **Add comment** iirc.